source/file_management/file_management.rst

```diff
@@ -140,27 +146,26 @@ and then save your new configuration.
 
 .. _fm-projects-folders:
 
-You can add and remove folders to a project
-with the **Project** menu
-and the side bar's context menu.
-Alternatively,
-you can drag a folder onto a window
-and it will be added automatically.
+You can add and remove folders to/from a project
```

Comment:
I suggest we avoid using slashes like this (re style guidelines) and instead do it like "to or from".